app/scripts/controllers/onboarding.js

```diff
@@ -62,6 +63,13 @@ export default class OnboardingController {
     this.store.updateState({ firstTimeFlowType: type });
   }
 
+  /**
+   * Setter for the `hasSetPassword` property
+   */
+  setHasSetPassword() {
```

Comment:
@darkwing is this needed? we should be able to determine whether they've set the password from the keyring controller right? if the password has been created, it should be in our state somewhere. Can we use that to determine whether the user has set the password?